stock_inventory_hierarchical_location/__openerp__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module is copyright (C) 2013 Numérigraphe SARL. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": "Exhaustive and hierarchical Stock Inventories",
+    "summary": "Extra consistency checks",
+    "version": "7.0.1.1.0",
```

Comment:
Yes, but this is not the module we're migrating in this PR. This is a side-effect of bringing the history in the repo. We need to bring 3 modules's history into the repo together because they have lots of common commits.

If you'd like to help us make it clearer, can you please review this PR before? https://github.com/OCA/stock-logistics-warehouse/pull/137
This way only the last commit would be shown here.